meson.build

```diff
@@ -1456,8 +1458,12 @@ conf.set10('HAVE_QRENCODE', have)
 
 want_gcrypt = get_option('gcrypt')
 if want_gcrypt != 'false' and not skip_deps
-        libgcrypt = cc.find_library('gcrypt', required : want_gcrypt == 'true')
-        libgpg_error = cc.find_library('gpg-error', required : want_gcrypt == 'true')
+        libgcrypt = dependency('libgcrypt', required : want_gcrypt == 'true')
```

Comment:
and this as well - are they really all supported from the exact same distribution version?

Reply:
Yes. All libraries are checked in Debian and Ubuntu package search page.